commcare_connect/opportunity/tables.py

```diff
@@ -58,15 +59,26 @@ def show_warning(record):
 
 class UserVisitReviewFilter(FilterSet):
     review_status = ChoiceFilter(choices=VisitReviewStatus.choices, empty_label="All Reviews")
+    user = ModelChoiceFilter(queryset=User.objects.none(), empty_label="All Users", to_field_name="username")
+    visit_date = DateRangeFilter()
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.filters["user"].queryset = User.objects.filter(id__in=self.queryset.values_list("user_id", flat=True))
+        self.filters["user"].field.label_from_instance = lambda obj: obj.name
+
         self.form.helper = FormHelper()
         self.form.helper.disable_csrf = True
         self.form.helper.form_class = "form-inline"
-        self.form.helper.layout = Layout(Row(Column("review_status", css_class="col-md-3")))
-
-        self.form.fields["review_status"].widget.attrs.update({"@change": "$refs.reviewFilterForm.submit()"})
+        self.form.helper.layout = Layout(
+            Row(
+                Column("review_status", css_class="col-md-3"),
+                Column("user", css_class="col-md-3"),
+                Column("visit_date", css_class="col-md-3"),
+            )
+        )
+        for field_name in self.form.fields.keys():
+            self.form.fields[field_name].widget.attrs.update({"@change": "$refs.reviewFilterForm.submit()"})
```

Comment:
does this submit the form or just refresh it?

Reply:
this submits the review filter form whenever any filter in that form is changed.

Reply:
ah filters are their own form?

Reply:
Yup. The filters are in their own form.